frontend/src/routes/(authenticated)/project/[project_code]/+page.svelte

```diff
@@ -176,7 +186,16 @@
     </div>
 
     <div class="divider" />
-
+    {#if isAdmin(user)}
+    <div class="space-y-2">
+      <p class="text-2xl mb-4">
+        <span class="link" target="_blank" on:click={() => resetProject()}>
+          {$t('project_page.reset_project_model.title', {name: project?.name})}
```

Comment:
At this point we know that `project` is defined, so you could use the non-nullable one. Not a great improvement. I realize now that it might be nice to move the content of the `{#if project}` condition into a component that only has a non-nullable `project`.
```suggestion
          {$t('project_page.reset_project_model.title', {name: _project.name})}
```

Reply:
Done.